### PR TITLE
disclosure examples: Support high contrast for graphics in Codepen by using current color in SVTG in CSS content

### PR DESCRIPTION
**Pull request by spectranaut**
This involves moving some PNG to SVGs in order to have the SVG inline with the CSS. We need to do this because CSS files cannot have relative paths that work correctly in Codepen (as codepen files do not have a file structure).

To review this PR, we need to review:
1. visual change of having an SVG instead of a PNG for the down and right arrows which indicate whether a node is expanded or collapsed.
2. That the "open in Codepen button" works correctly

[Preview Disclosure FAQ Example](https://raw.githack.com/w3c/aria-practices/SVG-move-png/examples/disclosure/disclosure-faq.html)

[Preview Disclosure Image Example](https://raw.githack.com/w3c/aria-practices/SVG-move-png/examples/disclosure/disclosure-img-long-description.html)

### CODE REVIEW

**Review by carmacleod — APPROVED** [review]
+1
Looks good - CodePen buttons work, including titles! :)

If you could change the fill and stroke for the SVGs to be 'currentColor', then they would pass in Windows High Contrast Mode.
I tested that currentColor works by changing the ::before content url's fill/stroke from #034575 to currentColor _**in the CodePen**_!! 🎉.
Let me know if you want me to open a separate issue for this.

**Comment by spectranaut** [comment]
@carmacleod I added `currentColor` to get it in :) thx for alerting me to it!

File: cspell.json [hunk 1]
```diff
@@ -34,7 +34,9 @@
         "Cook'n",
         "Copernicium",
         "Coughlin",
+        "Cpolygon",
```
Comment (carmacleod) [inline cspell.json:37]: don't need this after #1656 is merged

File: cspell.json [hunk 2]
```diff
@@ -34,7 +34,9 @@
...
         "crossorigin",
+        "Csvg",
```
Comment (carmacleod) [inline cspell.json:39]: don't need this after #1656 is merged

**Comment by mcking65** [comment]
Something is odd; looks like this PR is only changing to currentcolor for fill and stroke. Master already has the PNGs converted to SVG in CSS. I'll go ahead with it. I don't understand how the changes are already in master.

**Pull request merged**